lib/connection/result/column.js

```diff
@@ -561,7 +563,7 @@ function convertRawVariant(rawColumnValue, column, context)
       if (XMLValidator.validate(rawColumnValue) === true)
       {
         // use XML parser
-        ret = new XMLParser().parse(rawColumnValue);
+        ret = new XMLParser(xmlParserConfiguration).parse(rawColumnValue);
```

Comment:
do we want to pass it as it is? shouldn't we map it here to have separation what is our configuration and what is xml parser configuration?

Reply:
In this place it is merged.